docs/100_community/050_pepr-media.md

```diff
@@ -0,0 +1,24 @@
+# Pepr Media
+
+## 2024
+
+***Session Recordings***
+- [Optimizing Kubernetes Operators and Admission Control - Open Source Summit NA 2024](https://www.youtube.com/watch?v=SqJfvQ1frr4)
+- [Pepr Analyzation and Demo - YouTube](https://www.youtube.com/watch?v=kgSfS68hzbY)
```

Comment:
```suggestion
- [Pepr Analysis and Demo - YouTube](https://www.youtube.com/watch?v=kgSfS68hzbY)
```
Word-swap, cuz... I wasn't even sure "analyzation" was a real word until I looked it up. :see_no_evil:   I think the much-more-common-"analysis" will be familiar / easier to read for most readers, but... kind of a judgement call.  Feel free to ignore you don't like it.

From [Merriam-Webster](https://www.merriam-webster.com/dictionary/analyzation#usage-discussion-1-anchor)
```
Although the word analyzation has existed since the 18th century and is regularly formed from the verb analyze, it is much more rarely used than its more familiar synonym analysis.
```